.github/workflows/build-plugin-zip.yml

```diff
@@ -91,13 +122,17 @@ jobs:
                   sed -i "s/${{ steps.get_version.outputs.old_version }}/${VERSION}/g" gutenberg.php
                   sed -i "s/${{ steps.get_version.outputs.old_version }}/${VERSION}/g" readme.txt
 
-            - name: Commit the version bump
+            - name: Commit the version bump to the release branch
               run: |
                   git add gutenberg.php package.json package-lock.json readme.txt
                   git commit -m "Bump plugin version to ${{ steps.get_version.outputs.new_version }}"
                   git push --set-upstream origin "${{ steps.get_version.outputs.release_branch }}"
 
-            - name: Cherry-pick to trunk
+            - name: Fetch trunk
+              if: ${{ github.ref != 'refs/heads/trunk' }}
```

Comment:
I guess this is the important change right? Preventing the cherry-pick if we don't use trunk as a base branch?

Reply:
Ah no, not really, we were [doing that before](https://github.com/WordPress/gutenberg/pull/32560/files#diff-40b47d855e2cffe931290bba37c1b8716642627cbe05ac859728cb95c5da7536L104-L108) -- just using a different heuristic, i.e. by checking whether the version in `trunk`'s `package.json` matched the one from the branch the workflow was triggered from 😬

However, we [limited at the time which branches the workflow could be triggered to `trunk`](https://github.com/WordPress/gutenberg/pull/32560/files#diff-40b47d855e2cffe931290bba37c1b8716642627cbe05ac859728cb95c5da7536L27) only, so the check was a bit academic (and arguably a violation of the YAGNI pattern, that I hold so dear 😉).